src/utils/decorators/loggers/errorLog.ts

```diff
@@ -0,0 +1,24 @@
+export function errorLog() {
+  return function(target: any, propertyKey: string, descriptor: PropertyDescriptor){
+    const targetMethod = descriptor.value;
+
+    descriptor.value = async function (...args: any[]) {
+      const [req,,next] = args;
+      try {
+        const result = await targetMethod.apply(this, args);
+        if(result?.error) {
```

Comment:
if error is thrown from target funciton if would be catched in `catch` block, what case is handled by this if?

Reply:
Maybe I didn't understand correctly the task, but I response custom error not only from throw Error - I implemented errors catching in custom way (in simple response)